packages/cmf/src/sagaRouter/router.js

```diff
@@ -146,27 +154,33 @@ function parseSagaState(routeFragment, sagas, currentLocation) {
 export default function* sagaRouter(history, routes) {
 	const sagas = {};
 	const routeFragments = Object.keys(routes);
-	while (true) {  // eslint-disable-line no-constant-condition
+	while (true) {
+		// eslint-disable-line no-constant-condition
```

Comment:
this doesn't work, bad linter ?

Reply:
yes, it's committed by CI's prettier :disappointed: